CMakeLists.txt

```diff
@@ -137,25 +138,25 @@ IF (DOWNLOAD_TITLE_SEQUENCES)
     install(CODE "file(REMOVE \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/share/${PROJECT}/title/title-sequences.zip)")
```

Comment:
what about these here? especially the `execute_process` line above, that I cannot comment on with GitHub's UI?